<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>textTitle</class>
 <widget class="QDialog" name="textTitle">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>115</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>477</width>
    <height>115</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>477</width>
    <height>115</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>customGeoAttrib</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>3</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="toolTip">
          <string extracomment="escribir en minusculas"/>
         </property>
         <property name="text">
          <string>Nombre de especie (Escribir en minuscula)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
       <property name="leftMargin">
        <number>100</number>
       </property>
       <item>
        <widget class="QCheckBox" name="overWriteCheckBox">
         <property name="text">
          <string>sobreEscribir?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="process_Btn">
         <property name="text">
          <string>Process</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_btn">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
